shared-functions.sh

```diff
@@ -81,14 +81,14 @@ ensure_mac_os() {
 # Mac-specific function to install Java JDK
 install_mac_java() {
     # Determine which Java JDKs we have.
-    # If a non-Adopt Open JDK that is v1.8 (aka Java 8), that's ok!
-    java_versions=$(/usr/libexec/java_home --version "1.8" >/dev/null 2>&1 || echo "Not found")
+    # If a non-Adopt Open JDK that is v1.11 (aka Java 11), that's ok!
+    java_versions=$(/usr/libexec/java_home --version "1.11" >/dev/null 2>&1 || echo "Not found")
 
     if [ "$java_versions" = "Not found" ]; then
-        echo "Installing Adopt Open JDK v8..."
-        brew install homebrew/cask-versions/adoptopenjdk8
+        echo "Installing Adopt Open JDK v11..."
+        brew install homebrew/cask-versions/adoptopenjdk11
```

Comment:
Running `brew install homebrew/cask-versions/adoptopenjdk11` failed with

```
➜  webapp git:(course-editing-federation) ✗ brew install homebrew/cask-versions/adoptopenjdk11     
Warning: No available formula or cask with the name "homebrew/cask-versions/adoptopenjdk11".
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
```

But after `brew tap AdoptOpenJDK/openjdk`, I was able to `brew install adoptopenjdk11` successfully.

Reply:
Great, I have a fix for this ready to go.

Reply:
Ah! I dont see any changes here... Did you forget to push or should I be looking somewhere else?

Reply:
I haven\t pushed it yet, I wanted to do both at once.

I agree with you just reinstalling all the time makes sense.